src/common/components/search/SearchBar.tsx

```diff
@@ -120,26 +146,29 @@ const SearchBar: FC<SearchBarProps> = ({ shortcut }) => {
 
             <div className="relative block transition-all ">
               <span
-                onClick={() => setFocus('query', { shouldSelect: true })}
+                onClick={() => searchInputRef?.current?.focus()}
                 className="absolute inset-y-0 left-[45%] flex cursor-pointer items-center rounded-full p-3 shadow-lg shadow-black transition-all hover:scale-125  group-focus-within:left-0 group-focus-within:scale-100 group-focus-within:bg-transparent group-focus-within:shadow-none md:left-0"
               >
                 <Search className="h-6 w-6 text-white " aria-hidden="true" />
               </span>
 
-              <input
+              <DebounceInput
+                minLength={2}
+                debounceTimeout={300}
                 id="search"
                 autoComplete={`off`}
                 autoCorrect={`off`}
-                className="block w-full rounded-full border border-transparent bg-transparent py-3 pl-12 pr-2 text-base placeholder-transparent transition-all focus:border-white focus:placeholder-gray-500 focus:outline-none focus:ring-white sm:text-sm"
```

Comment:
Reason for more padding?

Reply:
it was a bit off with the hover effect of the search icon. I'll perfect it when I look at keyboard nav in the future